### PR TITLE
Fixes anti-adblock message on youtube.com

### PR DESCRIPTION
**Pull request by ryanbr**
Fixes anti-adblock used on youtube.com, 

Remove the bait:
`youtube.com,youtubekids.com,youtube-nocookie.com##+js(json-prune, [].playerResponse.adPlacements [].playerResponse.playerAds playerResponse.adPlacements playerResponse.playerAds adPlacements playerAds)`

While dismissiable the popup will reappear for the user every 5hrs or so. Also the popup prevents the video from loading.

Landed in ADG: https://github.com/AdguardTeam/AdguardFilters/commit/d6b0cf4d18e60b8a106d1c907625d662540c4cd1

### CODE REVIEW

**Comment by Yuki2718** [comment]
@gorhill Can we address this in main list? It's dismissable, but I saw a reddit post it reappears every 5 hours at minimum - can't find the post now tho.

**Comment by ryanbr** [comment]
For reference; https://www.bleepingcomputer.com/news/technology/youtube-tests-blocking-videos-unless-you-disable-ad-blockers/

**Comment by stephenhawk8054** [comment]
OP in this thread seems seeing the popup quite often?

https://www.reddit.com/r/uBlockOrigin/comments/13cdbjp/youtube_now_detects_ublock_and_idk_how_to_fix_it/jjr1vt4/

File: filters/annoyances.txt [hunk 1]
```diff
@@ -6831,6 +6831,10 @@ ph.ontools.net##+js(acs, document.oncontextmenu)
 ! https://github.com/uBlockOrigin/uAssets/issues/18048
 neilpatel.com##+js(no-xhr-if, /hotjar|googletagmanager/)
 
+! Youtube warning
+youtube.com##+js(json-prune, auxiliaryUi.messageRenderers.enforcementMessageViewModel)
+youtube.com##+js(set, ytInitialPlayerResponse.auxiliaryUi.messageRenderers.enforcementMessageViewModel, undefined)
```
Comment (Yuki2718) [inline filters/annoyances.txt:6836]: Why not `,youtubekids.com,youtube-nocookie.com`?

Comment (ryanbr) [reply filters/annoyances.txt:6836]: no one has reported these sites yet

**Comment by Yuki2718** [comment]
Unless @gorhill or other members oppose, I'm ready to merge. The rationale is we never purposefully block bait. If we found an once-useful specific rule is now bait, just remove it as obsolete regardless how it is used.

**Comment by gorhill** [comment]
I don't oppose, I just hope no unforeseen breakage given how Youtube is a high traffic site.

**Comment by stephenhawk8054** [comment]
Including

```adb
youtubekids.com,youtube-nocookie.com,youtube.com##+js(json-prune, [].playerResponse.adPlacements [].playerResponse.playerAds)
youtubekids.com,youtube-nocookie.com,youtube.com##+js(json-prune, playerResponse.adPlacements playerResponse.playerAds adPlacements playerAds, playerConfig)
```

?

**Comment by Yuki2718** [comment]
> You were able to reproduce anti-adblock ?

No, even with the set-constant rules in private discussion and old json-prune.

**Comment by Yuki2718** [comment]
> one of the rules was excluded

It was interfering with uBO's own json-prune rule.

**Comment by Yuki2718** [comment]
How about AG?

**Comment by AdamWr** [comment]
I have added this rule.

**Comment by Yuki2718** [comment]
> need to remove the duplicates from quick fixes list.

What duplicates?

> How did it end up in annoyances?

Have you read the comments here?

**Comment by Yuki2718** [comment]
My point was it should be fixed in Annoyances as per https://github.com/uBlockOrigin/uAssets/pull/18106#issuecomment-1551302507 but not blocking bait is another thing, so splitting the fixes as suggested in https://github.com/uBlockOrigin/uAssets/pull/18106#issuecomment-1551312691

**Pull request merged**